### DISCUSSION TITLE
Is there a way to change the color of the top bar where the search bar is located at?

### DISCUSSION THREAD

**Discussion by Pazifix** [aaFn/Bookmark-search-plus-2#302] [Q&A]
Is there a way to change the color of the top bar where the search bar is located at?

**Comment by aaFn** [comment]
Hello @Pazifix, you can change the color of the background of the panel, by aligning with the FF theme or by specifying your own background color, but not the color inside the search box.

**Comment by Gitoffthelawn** [comment]
@Pazifix 
If you are using Firefox, you can create a userContent.css file, and customize BSP2 to your heart's desire! If you're not familiar with userContent.css, where to place it, or which Firefox prefs need to be set to use it, a web search will yield comprehensive details.
